app/services/course_search_service.rb

```diff
@@ -10,8 +12,12 @@ def initialize(
   end
 
   class << self
-    def call(...)
-      new(...).call
+    def provider_ascending
```

Comment:
This seems a little redundant as class methods. Seems to only be used internally? They're also immutable - feels like they would fit better as constants in the class?

Reply:
Good call done in https://github.com/DFE-Digital/publish-teacher-training/pull/3157/commits/d8724089fc9f5e26c3c7c589b83dc7b8a600452d